packaging/datadog-agent/source/install_agent.sh

```diff
@@ -144,7 +144,11 @@ else
 fi
 
 printf "\033[34m* Starting the Agent...\n\033[0m\n"
-$sudo_cmd /etc/init.d/datadog-agent restart
+if [ $OS = "Debian" ]; then
+    $sudo_cmd invoke-rc.d datadog-agent restart
+else
+    $sudo_cmd /etc/init.d/datadog-agent restart
+fi
```

Comment:
Should we use the same test that we use in the postinst script? If `invoke-rc.d` is present use it, otherwise fallback on the init script.